### PR TITLE
Link models linked node appearance

### PR DESCRIPTION
**Pull request by sebeder**
[Trello Card](https://trello.com/c/m5T8PfWo/662-kapselung-optische-darstellung-verlinkter-knoten)

### CODE REVIEW

**Review by wehrlean — CHANGES_REQUESTED** [review]
1. With double clicking the text is still editable
2. And if you rest with the cursor on the 🔗 symbol, you get a hint with the same symbol. No direct error but not necessary
3. On reload, the nodes are small and on the left top corner
image

File: web/src/app/modules/views/main/editors/modules/graphical-editor/providers/properties/vertex-provider.ts [hunk 1]
```diff
@@ -68,15 +68,20 @@ export class VertexProvider extends ProviderBase {
         this.graph.getModel().beginUpdate();
         const vertex = this.graph.insertVertex(parent, url, value, x, y, width, height, style);
         const vertexVariable = this.graph.insertVertex(vertex, url + VertexProvider.ID_SUFFIX_VARIABLE, data.variable,
-            VertexProvider.INITIAL_CHILD_NODE_X, 0.15, 0, (mx.mxConstants.DEFAULT_FONTSIZE), EditorStyle.VARIABLE_NAME_STYLE, true);
+            // tslint:disable-next-line: max-line-length
+            VertexProvider.INITIAL_CHILD_NODE_X, 0.15, 0, (mx.mxConstants.DEFAULT_FONTSIZE), EditorStyle.VARIABLE_NAME_DISABLED_STYLE, true);
         const vertexCondition = this.graph.insertVertex(vertex, url + VertexProvider.ID_SUFFIX_CONDITION, data.condition,
-            VertexProvider.INITIAL_CHILD_NODE_X, 0.4, 0, (mx.mxConstants.DEFAULT_FONTSIZE), EditorStyle.TEXT_INPUT_STYLE, true);
+            VertexProvider.INITIAL_CHILD_NODE_X, 0.4, 0, (mx.mxConstants.DEFAULT_FONTSIZE), EditorStyle.TEXT_INPUT_DISABLED_STYLE, true);
+        const vertexSymbol = this.graph.insertVertex(vertex, url + VertexProvider.ID_SUFFIX_CONDITION, '🔗',
```
Comment (wehrlean) [inline web/src/app/modules/views/main/editors/modules/graphical-editor/providers/properties/vertex-provider.ts]: VertexProvider.ID_SUFFIX_CONDITION is not correct for the symbol

**Review by wehrlean — CHANGES_REQUESTED** [review]

**Comment by sebeder** [comment]
OMG! I'll resolve the issues now... Coding in the bus is not my strength, obviously.

**Comment by wehrlean** [comment]
> 
> 
>     1. With double clicking the text is still editable
> 
>     2. And if you rest with the cursor on the 🔗 symbol, you get a hint with the same symbol. No direct error but not necessary
> 
>     3. On reload, the nodes are small and on the left top corner
>        image

Still all point not solved

**Comment by sebeder** [comment]
Should work now... Was a little bit harder than it seemed.

**Review by wehrlean — APPROVED** [review]
Now everything is fine :)

**Pull request merged**